config.go

```diff
@@ -684,17 +688,24 @@ func loadConfig() (*config, []string, error) {
 		}
 	}
 
-	if cfg.RPCConnect == "" {
-		cfg.RPCConnect = net.JoinHostPort("localhost", activeNet.RPCClientPort)
+	// Validate extended vote bits
+	voteBitsExtendedB, err := hex.DecodeString(cfg.VoteBitsExtended)
+	if err != nil {
+		err = fmt.Errorf("invalid votebitsextended setting: %v", err)
+		fmt.Fprintln(os.Stderr, err.Error())
+		fmt.Fprintln(os.Stderr, usageMessage)
+		return loadConfigError(err)
 	}
-
-	// Set ticketfee and txfee to defaults if none are set.  Avoiding using default
-	// confs because they are dcrutil.Amounts
-	if cfg.TicketFee == 0.0 {
-		cfg.TicketFee = wallet.DefaultTicketFeeIncrement.ToCoin()
+	if len(voteBitsExtendedB) > stake.MaxSingleBytePushLength-2 {
```

Comment:
still unsure about this. I think we need to enforce that it is some exact length